rts/Sim/Units/UnitHandler.cpp

```diff
@@ -351,16 +355,42 @@ void CUnitHandler::SlowUpdateUnits()
 	const size_t idxEnd = idxBeg + indCnt;
 
 	activeSlowUpdateUnit = idxEnd;
-
 	// stagger the SlowUpdate's
-	for (size_t i = idxBeg; i<idxEnd; ++i) {
-		CUnit* unit = activeUnits[i];
 
-		unit->SanityCheck();
-		unit->SlowUpdate();
-		unit->SlowUpdateWeapons();
-		unit->localModel.UpdateBoundingVolume();
-		unit->SanityCheck();
+	int slowUpdateBoundingVolumeMT = configHandler->GetInt("SlowUpdateBoundingVolumeMT");
+	if (slowUpdateBoundingVolumeMT == 0) {
+		for (size_t i = idxBeg; i < idxEnd; ++i) {
+			CUnit* unit = activeUnits[i];
+
+			unit->SanityCheck();
+			unit->SlowUpdate();
+			unit->SlowUpdateWeapons();				
+			unit->localModel.UpdateBoundingVolume();
+			unit->SanityCheck();
+		}
+	}
+	else {
+		{
+			ZoneScopedN("Sim::Unit::SlowUpdateST");
+			for (size_t i = idxBeg; i < idxEnd; ++i) {
+				CUnit* unit = activeUnits[i];
+
+				unit->SanityCheck();
```

Comment:
Does it need this sanity check if there's also one in the MT section?

Reply:
IIRC that sanity check compiles to a no-op on release. I think its thus not a bad idea to have it both before and after slowupdate, similarly to ST implementation